OrbitGl/Batcher.cpp

```diff
@@ -220,17 +225,28 @@ void Batcher::GetBoxGradientColors(const Color& color, std::array<Color, 4>* col
 }
 
 void Batcher::ResetElements() {
-  line_buffer_.Reset();
-  box_buffer_.Reset();
-  triangle_buffer_.Reset();
+  for (auto& [_, buffer] : buffers_by_layer_) {
+    buffer.line_buffer.Reset();
```

Comment:
Consider defining a Reset() method on the Buffers type so you don't have to call all three Reset methods here (which is a minor inconvenience, but potentially a bit more error prone).